schemas/v1.0/schema.yaml

```diff
@@ -0,0 +1,563 @@
+        description: The stepId to transfer to upon success of the step
+        $dynamicRef: '#stepId'
+      criteria:
+        description: A list of assertions to determine if this action SHALL be executed
+        type: array
+        uniqueItems: true
+        minItems: 1
+        items:
+          $ref: '#/$defs/criterion-object'
+    allOf:
+      - if:
+          properties:
+            type:
+              const: goto
+        then:
+          oneOf:
+            - required:
+                - workflowId
+            - required:
+                - stepId
+    required:
+      - name
+      - type
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  failure-action-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#failure-action-object
+    description: A single failure action which describes an action to take upon failure of a workflow step
+    type: object
+    properties:
+      name:
+        description: The name of the failure action
+        type: string
+      type:
+        description: The type of action to take
+        enum:
+          - end
+          - goto
+          - retry
+      workflowId:
+        description: The workflowId referencing an existing workflow within the Arazzo description to transfer to upon failure of the step
+        $dynamicRef: '#workflowId'
+      stepId:
+        description: The stepId to transfer to upon failure of the step
+        $dynamicRef: '#stepId'
+      retryAfter:
+        description: A non-negative decimal indicating the seconds to delay after the step failure before another attempt SHALL be made
+        type: number
+        minimum: 0
+      retryLimit:
+        description: A non-negative integer indicating how many attempts to retry the step MAY be attempted before failing the overall step
+        type: integer
+        minimum: 0
+      criteria:
+        description: A list of assertions to determine if this action SHALL be executed
+        type: array
+        uniqueItems: true
+        items:
+          $ref: '#/$defs/criterion-object'
+    allOf:
+      - if:
+          properties:
+            type:
+              enum:
+                - goto
+                - retry
+        then:
+          oneOf:
+            - required:
+                - workflowId
+            - required:
+                - stepId
+      - if:
+          properties:
+            type:
+              const: retry
+        then:
+          required:
+            - retryAfter
+    required:
+      - name
+      - type
+    $ref: '#/$defs/specification-extensions'
+    unevaluatedProperties: false
+  reusable-object:
+    $comment: https://spec.openapis.org/arazzo/v1.0.0#reusable-object
+    description: A simple object to allow referencing of objects contained within the Components Object
+    type: object
+    properties:
+      reference:
+        description: A runtime expression used to reference the desired object
+        type: string
+      value:
```

Comment:
In spec this value is `string`

```
value | string | Sets a value of the referenced parameter. This is only applicable for parameter object references.
-- | -- | --
```

https://spec.openapis.org/arazzo/latest.html#fixed-fields-8

Reply:
@DmitryAnansky FYI - There's an open issue for this https://github.com/OAI/Arazzo-Specification/issues/231 as it should really match what is specified at [4.6.6.1Fixed Fields](https://spec.openapis.org/arazzo/latest.html#fixed-fields-4) for `value`